source/PlanetPanel.cpp

```diff
@@ -319,6 +319,14 @@ void PlanetPanel::CheckWarningsAndTakeOff()
 	int missionCargoToSell = cargo.MissionCargoSize() - cargo.Size();
 	// Will you have to sell something other than regular cargo?
 	int cargoToSell = -(cargo.Free() + cargo.CommoditiesSize());
+	// Have you left any unique items at the outfitter?
+	bool hasUniques = false;
```

Comment:
Comment and variable name seem to say different things, after that comment I would expect something like "leftUniques"